gradle.properties

```diff
@@ -1,4 +1,4 @@
-VERSION_NAME=3.3.1-SNAPSHOT
+VERSION_NAME=4.0.0-SNAPSHOT
```

Comment:
This looks like a bug fix that is not breaking or changing any public API. I dont think a major version update is required

Reply:
It is a breaking change since the signature for the **public** method _LangUtils.setAppLocale(_ changes